Classes/RZPoseurWebView.m

```diff
@@ -71,6 +71,12 @@ - (NSURLRequest *)request
     return nil;
 }
 
+- (UIScrollView *)scrollView
+{
+    RZPoseurWebViewMethodUnimplementedAssert();
+    return nil;
```

Comment:
Thanks for explaining this offline. `backingView` is implemented by each subclass independently. We can adopt `forwardInvocation` or some other less duplicative approach down the road.